test/src/components/range_selector/range_selector_test.dart

```diff
@@ -0,0 +1,509 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:zeta_flutter/zeta_flutter.dart';
+
+import '../../../test_utils/test_app.dart';
+import '../../../test_utils/tolerant_comparator.dart';
+import '../../../test_utils/utils.dart';
+
+void main() {
+  const String parentFolder = 'range_selector';
+
+  const goldenFile = GoldenFiles(component: parentFolder);
+  setUpAll(() {
+    goldenFileComparator = TolerantComparator(goldenFile.uri, tolerance: 0.001);
```

Comment:
why are you using a different tolerance?

Reply:
That is interesting. We should experiment with other components to see if more can have a lower tolerance

Reply:
I think it's more of a case of if certain components Should have a lower tolerance due to there design. For example if the component takes up less area in the 800x600 plane,,,, say less than 10% of the pixels (just an example) then the tolerance should be lowered as small pixel changes may not even be a 1% difference

Reply:
I guess we should write a function to ensure the components take up the whole space in the tests